### ISSUE TITLE
AutoComplete: Accessibility Improvement

### ISSUE THREAD

**Issue by EmirBoyaci** [primefaces/primereact#3275]
### Describe the feature you would like to see added

Hello,

Since accessibility is a important and a blocker for people with disabilities, I would like to propose adding accessibility attribute(s) to the AutoComplete. Actually, it is already thought by someone (see: [AutoComplete:481](https://github.com/primefaces/primereact/blob/master/components/lib/autocomplete/AutoComplete.js#L481)) and added some aria attributes to the AutoComplete component. I don't know is it forgotten but AutoComplete does not allow to pass aria attributes to the Button component which is rendered when you pass `dropdown={true}`. Therefore, we cannot pass aria attributes and when I generate a Lighthouse report it says "Buttons do not have an accessible name". Also, in Accessibility tool inside Chrome's developer tools, we can see that `aria-label` is not specified.

**Lighthouse:**

![lighthouse](https://user-images.githubusercontent.com/59085992/189427238-4c7cdb60-cecb-4145-9f5f-20d30d4a6fda.png)

**Chrome Developer Tools:**

![chrome-devtools](https://user-images.githubusercontent.com/59085992/189427251-b9736fde-6545-407e-81d1-e5dccdc8b7ab.png)

### Is your feature request related to a problem?

This feature request is improves accessibility for people with disabilities.

### Describe the solution you'd like

Adding a prop which allows developers to pass aria attributes to the AutoComplete > Button component can solve this feature request.

### Additional context

https://dequeuniversity.com/rules/axe/4.4/button-name

**Comment by melloware** [comment]
I just fixed something similar: https://github.com/primefaces/primereact/issues/3259  let me look

**Comment by melloware** [comment]
OK i added `dropdownAriaLabel` but if you don't add that I will try and use `placeholder` value and then fall back to Locale "choose" option so its always filled out with at least something proper.

**Comment by EmirBoyaci** [comment]
> OK i added `dropdownAriaLabel` but if you don't add that I will try and use `placeholder` value and then fall back to Locale "choose" option so its always filled out with at least something proper.
> 
> cc @kyybo

Thanks for your quick implementation and efforts!
